channeldb/graph.go

```diff
@@ -1387,16 +1387,24 @@ func putChanEdgeInfo(edgeIndex *bolt.Bucket, edgeInfo *ChannelEdgeInfo, chanID [
 	}
 
 	authProof := edgeInfo.AuthProof
-	if err := wire.WriteVarBytes(&b, 0, authProof.NodeSig1.Serialize()); err != nil {
+	var nodeSig1, nodeSig2, bitcoinSig1, bitcoinSig2 []byte
+	if authProof != nil {
```

Comment:
As we discussed offline the signatures/proof might be equal to nil in the case if we don't want to announce the channel.